src/app/containers/Article/index.jsx

```diff
@@ -11,6 +12,20 @@ import MainContent from '../../components/MainContent';
 import articlePropTypes from '../../models/propTypes/article';
 import { ServiceContextProvider } from '../../components/ServiceContext';
 import Timestamp from '../../components/Timestamp';
+import { layoutGridWrapper, layoutGridItem } from '../../lib/layoutGrid';
+import { C_OAT_LHT } from '../../lib/constants/styles';
+
+const Wrapper = styled.div`
+  ${layoutGridWrapper};
+`;
+
+const OatWrapper = styled(Wrapper)`
+  background: ${C_OAT_LHT};
+`;
+
+const Item = styled.div`
```

Comment:
To me this seems a bit of a vague naming - maybe better as `GridItem` or `ConstrainedItem`?

Reply:
Agreed. `GridItemConstrained` for this one, so we can later have a `GridItem` that is full-width. I will also change the variable in the layoutGrid file to be called `layoutGridItemConstrained`, for consistency.